From jejb@kernel.org  Tue Nov  4 11:35:00 2008
From: Scott James Remnant <scott@canonical.com>
Date: Thu, 30 Oct 2008 19:10:04 GMT
Subject: ipmi: add MODULE_ALIAS to load ipmi_devintf with ipmi_si
To: jejb@kernel.org, stable@kernel.org
Message-ID: <200810301910.m9UJA4GB012904@hera.kernel.org>

From: Scott James Remnant <scott@canonical.com>

commit 6c89161b10f5771ee0b51ada0fce0e8835e72ade upstream

The ipmi_devintf module contains the userspace interface for IPMI devices,
yet will not be loaded automatically with a system interface handler
driver.

Add a MODULE_ALIAS for the "platform:ipmi_si" MODALIAS exported by the
ipmi_si driver, so that userspace knows of the recommendation.

Signed-off-by: Scott James Remnant <scott@ubuntu.com>
Cc: Tim Gardner <tcanonical@tpi.com>
Cc: Corey Minyard <minyard@acm.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/char/ipmi/ipmi_devintf.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/char/ipmi/ipmi_devintf.c
+++ b/drivers/char/ipmi/ipmi_devintf.c
@@ -957,3 +957,4 @@ module_exit(cleanup_ipmi);
 MODULE_LICENSE("GPL");
 MODULE_AUTHOR("Corey Minyard <minyard@mvista.com>");
 MODULE_DESCRIPTION("Linux device interface for the IPMI message handler.");
+MODULE_ALIAS("platform:ipmi_si");
